### DISCUSSION TITLE
PlayStation Plus Essential games

### DISCUSSION THREAD

**Discussion by OJ7** [vogler/free-games-claimer#107] [Ideas]
I see Xbox Games with Gold is planned.  It would be nice to have an auto-claimer for PS Plus monthly games as well.

This link can be used for the current month's batch:
https://www.playstation.com/en-us/ps-plus/whats-new/#monthly-games

**Comment by vogler** [comment]
Can't implement it since I don't have a PlayStation and therefore no PS Plus.
You can claim the current games via their website when you're logged in?

**Reply by OJ7** [reply]
Correct, it shows up like this:

<img width="1792" alt="image" src="https://user-images.githubusercontent.com/5333243/227316711-2a8f2a35-689d-479b-993c-99e9bd9b023a.png">

**Reply by vogler** [reply]
You can copy `gog.js` which is the simplest script, remove the automated login code (can always login manually), and then play around with 'right click > Inspect' in your browser to find some selector to click the button.

**Reply by OJ7** [reply]
I've got an initial working version of this here: https://github.com/vogler/free-games-claimer/compare/main...OJ7:free-games-claimer:psn

Still has some bugs with previous login state not always being correct (the playstation website is weird and sometimes requires clicking sign in a few times), but re-running the script a couple times should allow it to go through.

Looking for some testers to verify functionality (especially since I've already claimed games for the month).

**Reply by vogler** [reply]
I've linked your comment in #141.

**Reply by mfuery786** [reply]
Hi is there a plugin or script to claim all add ons and free content on ps plus android app or website without having to manually claim each free dlc/add-on or game
